src/main/java/com/google/cloud/tools/managedcloudsdk/install/Downloader.java

```diff
@@ -71,7 +72,9 @@ public void download() throws IOException, InterruptedException {
               Files.newOutputStream(destinationFile, StandardOpenOption.CREATE_NEW))) {
 
         progressListener.start(
-            "Downloading " + String.valueOf(contentLength) + " bytes", contentLength);
+            String.format(
+                Locale.getDefault(), "Downloading %.2f MB", contentLength / 1024.0f / 1024.0f),
```

Comment:
[java.util.Formatter](https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html)'s javadoc says you need to have the comma flag to get the numeric grouping: `%,.2f`.

Reply:
And this bug would be caught with a test that used a string literal showing the desired output rather than a test that duplicates the incomplete logic.